apps/examples/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
 	"extends": "../../config/tsconfig.base.json",
-	"include": ["src", "e2e", "./vite.config.ts"],
+	"include": ["src", "e2e", "./vite.config.ts", "**/*.json"],
```

Comment:
Probably something to remove later, if the typechecker spends too much time parsing json files.